R/api_key.R

```diff
@@ -0,0 +1,14 @@
+#' @export
+set_wpp_api_key <- function(key) {
+  Sys.setenv(WPP_API_KEY = key)
+}
+
+check_wpp_api_key <- function() {
+  if (Sys.getenv("WPP_API_KEY") == "") {
+    cli::cli_abort("No API key detected. Set using {.code set_wpp_key_key()} or inside `.Renviron`")
+  }
+}
+
+get_wpp_api_key <- function() {
+  Sys.getenv("WPP_API_KEY")
+}
```

Comment:
Should we do a `check_wpp_api_key` before explicitly returning the env variable?